cmd/fastly-exporter/main.go

```diff
@@ -275,6 +280,13 @@ func main() {
 			}
 			return nil
 		})
+		g.Go(func() error {
+			if err := productCache.Fetch(context.Background()); err != nil {
```

Comment:
Should we rename Fetch to Refresh to be consistent with the other caches?

Reply:
Yeah, that makes sense. I started with Fetch since it was only going to run once but `Refresh` is better for consistency.